ChangeLog.d/remove-rsa-mode-parameter.txt

```diff
@@ -0,0 +1,15 @@
+Removals
+   * The RSA module no longer supports private-key operations with the public
+     key and vice versa. This change only affects applications which use the
+     wrong mode. In this case the wrong mode is to use mode=MBEDTLS_RSA_PUBLIC
+     with decryption and signing functions and mode=MBEDTLS_RSA_PRIVATE with
+     encryption and verification functions. Addresses issue #4278.
+API changes
+   * Remove mode parameter from RSA functions. All encryption,
```

Comment:
Please write complete English sentences, not headlinespeak: “Remove the mode parameter …”, “The RNG is now mandatory …”, etc.